masterkey.js

```diff
@@ -0,0 +1,83 @@
+var https = require('http');
```

Comment:
This file is currently forked as it's not that big, currently has hardcoded config and we educate about executing `node masterkey.js` in the documentation repo. We could definitely look into providing parameters for a shared script, but I wasn't sure if it's worth the effort currently.